services/simcore/template.env

```diff
@@ -322,3 +321,5 @@ CLUSTERS_KEEPER_EC2_INSTANCES_SUBNET_ID=${CLUSTERS_KEEPER_EC2_INSTANCES_SUBNET_I
 CLUSTERS_KEEPER_EC2_REGION_NAME=${CLUSTERS_KEEPER_EC2_REGION_NAME}
 CLUSTERS_KEEPER_EC2_SECRET_ACCESS_KEY=${CLUSTERS_KEEPER_EC2_SECRET_ACCESS_KEY}
 CLUSTERS_KEEPER_LOGLEVEL=${CLUSTERS_KEEPER_LOGLEVEL}
+
+WORKERS_EC2_INSTANCES_TIME_BEFORE_TERMINATION=${WORKERS_EC2_INSTANCES_TIME_BEFORE_TERMINATION}
```

Comment:
question, so these template.env are useless... I think you  should remove anything that just add noise